api-server/docker-build.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
```

Comment:
I believer your github action scripts rely on this so we would have to change those to move away from this